Orange/widgets/data/oweditdomain.py

```diff
@@ -1613,6 +1838,7 @@ def __init__(self):
     @Inputs.data
     def set_data(self, data):
         """Set input dataset."""
+        self._merge_dialog_settings = self._editor.get_merge_context()
```

Comment:
This will not save the context when the widget is closed. I believe the proper solution is to define `storeSpecificSettings`:

```python
    def storeSpecificSettings(self):
        self._merge_dialog_settings = self._editor.get_merge_context()
```

Reply:
Thank you, I didn't think about that situation. It is fixed now.